guides/set-current-user.md

```diff
@@ -0,0 +1,97 @@
+[Back to Main README](https://github.com/simplabs/ember-simple-auth/blob/master/README.md)
+
+## How to Set currentUser Data
+
+Although there are various ways to load currentUser data, this is the canonical
+way to do so. Your authentication route will need to identify how it can access
+the data for the current user. In this example, it is sent as the user_id on the
+authentication payload.
```

Comment:
I think we should also include the scenario where instead of having the actual user id at hand you have a dedicated endpoint for loading the user record that belongs to the token (or whatever authentication info you have), e.g. `GET /users/me`. This would actually be what I'd advise most users to use as e.g. Auth 2.0 defines the response pretty strictly and wouldn't allow an additional `user_id` in the response at all.